### DISCUSSION TITLE
Content delivery to inbox - SMS, Messenger, Instagram DM and Whatsapp

### DISCUSSION THREAD

**Discussion by firozemzr** [OSBDAlliance/headlesscms#10] [Ideas]
CMS should have the ability to deliver content to selected user base in different channel e.g. SMS, Messenger, Instagram DM and Whatsapp. Relevant account credentials of different platforms should be easily configurable.

**Comment by mohammad-arif-rahim** [comment]
This is a presentation layer requirement. Even if the headless CMS provides API to integrate with the social media's integration will have to be the role of the deployer.

**Reply by firozemzr** [reply]
No this is not a presentation layer requirement.. This is a total back end function that calls api to deliver these messages.. Mostly either json payload or url parameters (for sms)

For CMS this is very critical back end function specially for subscription contents.
